src/components/modal/base/modal-title.jsx

```diff
@@ -0,0 +1,18 @@
+const PropTypes = require('prop-types');
+const React = require('react');
+
+require('./modal-title.scss');
+
+const ModalTitle = ({
+    title
+}) => (
+    <div className="modal-title">
```

Comment:
Since it's likely we might want to further modify the styles, it would be prudent to include a `className` prop (which would be added in addition to `modal-title`).

Reply:
I agree, and I had that lined up for the next PR, but I'm happy to put it into this one!